### ISSUE TITLE
Warning: Unknown warning kind unsound found, please, file a bug

### ISSUE THREAD

**Issue by FabianLars** [rustsec/audit-check#8]
First of all, thanks for taking over the action ❤️ 

We started seeing this warning in [CI](https://github.com/tauri-apps/plugins-workspace/actions/runs/4578790378/jobs/8085834440?pr=294#step:3:14) and i guess it's caused by https://github.com/rustsec/rustsec/pull/819 ?
I tried to fix it myself but i have no knowledge about cargo-audit nor the action so that didn't end well 😅 

Thanks again :)

**Comment by pinkforest** [comment]
Hey thanks! - Could we try there with `rustsec/audit-check@main` now as I added the correct informational types ?

I'll push our a new release for @v1.4.0 but would like to confirm here as well please

**Comment by FabianLars** [comment]
Yep seems to work, not seeing that warning anymore: https://github.com/tauri-apps/plugins-workspace/actions/runs/4605104520/jobs/8136735767 Thank you!

**Comment by pinkforest** [comment]
I've released v1.4.0 which can be used via `rustsec/audit-check@v1.4.0`

**Comment by pinkforest** [comment]
@FabianLars The behaviour when the workflow is triggered via cron is different so I had to fix that as well

I've corrected the scheduled workflow in v1.4.1 and now it should raise issues for unsound advisories as well.

**Comment by FabianLars** [comment]
Hello again. I don't know if i'm missing something obvious here, but i think that either i was wrong when i tested 1.4.0, or 1.4.1 re-introduced the issue since i'm seeing the warning log in PR checks again: https://github.com/tauri-apps/plugins-workspace/actions/runs/4721568327/jobs/8374994844?pr=309#step:3:14 (using `rustsec/audit-check@v1`)
